tracing/tck/src/main/java/org/eclipse/microprofile/telemetry/tracing/tck/async/JaxRsServerAsyncTest.java

```diff
@@ -119,8 +113,6 @@ private void doAsyncTest(Function<JaxRsServerAsyncTestEndpointClient, String> re
             }
         }
 
-        List<SpanData> spanData = spanExporter.getFinishedSpanItems(3);
-
```

Comment:
This shouldn't be removed.

Although I see that `spanData` isn't used, the call to `spanExporter.getFinishedSpanItems(3);` makes sure the expected number of spans are created before proceeding.